packages/sidecar/package.json

```diff
@@ -25,6 +25,9 @@
     "./vite-plugin": {
       "import": "./src/vite-plugin.js"
     },
+    "./connect": {
```

Comment:
is there no other sane way to define these main exports e.g. in like an index.ts or something?